source/robotstxtparser.php

```diff
@@ -104,6 +108,10 @@ public function __construct($content, $encoding = self::DEFAULT_ENCODING)
 
         // parse rules - default state
         $this->prepareRules();
+        
+        $this->regExpPatternReplace = empty($regExpPatternReplace)
+            ? array('@' => '\@')
```

Comment:
Are you sure you would want to replace it but not merge? Could you please provide a test for this case?